packer/windows-vhd-builder.json

```diff
@@ -0,0 +1,82 @@
+{
+    "variables": {
+        "client_id": "{{env `AZURE_CLIENT_ID`}}",
+        "client_secret": "{{env `AZURE_CLIENT_SECRET`}}",
+        "tenant_id": "{{env `AZURE_TENANT_ID`}}",
+        "subscription_id": "{{env `AZURE_SUBSCRIPTION_ID`}}",
+        "location": "{{env `AZURE_LOCATION`}}",
+        "vm_size": "{{env `AZURE_VM_SIZE`}}"
+    },
+    "builders": [
+        {
+            "type": "azure-arm",
+            "client_id": "{{user `client_id`}}",
+            "client_secret": "{{user `client_secret`}}",
+            "tenant_id": "{{user `tenant_id`}}",
+            "subscription_id": "{{user `subscription_id`}}",
+            "location": "{{user `location`}}",
+            "vm_size": "{{user `vm_size`}}",
+            "os_type": "Windows",
+            "image_publisher": "MicrosoftWindowsServer",
+            "image_offer": "WindowsServer",
+            "image_sku": "2019-Datacenter-Core-smalldisk",
+            "image_version": "17763.615.1907121548",
+            "resource_group_name": "{{user `resource_group_name`}}",
+            "capture_container_name": "aksengine-vhds-windows-ws2019",
+            "capture_name_prefix": "aksengine-{{timestamp}}",
+            "storage_account": "{{user `storage_account_name`}}",
+            "communicator": "winrm",
+            "winrm_use_ssl": true,
+            "winrm_insecure": true,
+            "winrm_timeout": "10m",
+            "winrm_username": "packer"

```

Comment:
does the `sysprep /generalize` remove this user and disable winrm later?

Reply:
The user is removed.
Looks like WinRM is still enabled but is cleaned up some.

Reply:
It looks like winrm is in the same state as a vanilla intall of ws2019-servercore-smalldisk in azure.

Reply:
I found a winrm listener that was left behind.
I updated the packer job to purge the listeners right before shutdown and it it looks like that cleaned everything up.